nevergrad/benchmark/experiments.py

```diff
@@ -21,6 +21,7 @@
 from nevergrad.functions.arcoating import ARCoating
 from nevergrad.functions import images as imagesxp
 from nevergrad.functions.powersystems import PowerSystem
+from nevergrad.functions.ac import Ac
```

Comment:
can you please rename to something more intelligible? AquaCrop seems ok, NgAquaCrop to differentiate it from the base one. AC is just not clear

Reply:
Fixed.